index.js

```diff
@@ -0,0 +1,39 @@
+const validateLineup = (lineup) => {
+  function getSalaries(lineup) {
+    return lineup.salary
+  }
+  function getSalarySum(acc, n) {
+    return acc + n
+  }
+  function getTeamId(lineup) {
+    return lineup.teamId
+  }
+  function getGameId(lineup) {
+    return lineup.gameId
+  }
+  function getPositions(lineup) {
+    return lineup.position
+  }
```

Comment:
Declaring named functions inside another function? These should live on their own.